src/FondOfSpryker/Zed/CompanyUserCompanyAssigner/Dependency/Client/CompanyUserCompanyAssignerToCompanyFacadeInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace FondOfSpryker\Zed\CompanyUserCompanyAssigner\Dependency\Client;
```

Comment:
@SpiderDan98 the namespace is not correct.

FondOfSpryker\Zed\CompanyUserCompanyAssigner\Dependency\Facade